ruby_version_file_parser_test.go

```diff
@@ -0,0 +1,101 @@
+package bundleinstall_test
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	bundleinstall "github.com/paketo-buildpacks/bundle-install"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+)
+
+const RUBY_VERSION_FILE_TEMPLATE = "%s\n"
```

Comment:
I don't think this needs to be a constant - it's small enough (and obvious enough) to inline where needed. I think elevating it to a constant is actually more confusing.